api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderConstants.java

```diff
@@ -382,6 +383,7 @@
             Extensions.PRIMARY_CARE_ADULT_FOLLOWUP_VISIT_ACTION,
             Extensions.PRIMARY_CARE_PEDS_INITIAL_VISIT_ACTION,
             Extensions.PRIMARY_CARE_PEDS_FOLLOWUP_VISIT_ACTION,
+			Extensions.NURSE_CONSULT_NOTE_VISIT_ACTION,
```

Comment:
This list lets us sort where the nurse consult action would appear amongst the list of visit actions